sdk/webpubsub/azure-messaging-webpubsubservice/tests/test_smoke.py

```diff
@@ -0,0 +1,28 @@
+# coding: utf-8
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# -------------------------------------------------------------------------
+from testcase import WebpubsubTest, WebpubsubPowerShellPreparer
+from azure.messaging.webpubsubservice.operations._operations import build_send_to_all_request
+
+class WebpubsubSmokeTest(WebpubsubTest):
+
+
+    @WebpubsubPowerShellPreparer()
+    def test_webpubsub_send_to_all(self, webpubsub_endpoint):
+        client = self.create_client(endpoint=webpubsub_endpoint)
+        client.send_to_all('Hub', {'hello': 'test_webpubsub_send_to_all'})
+
+    @WebpubsubPowerShellPreparer()
+    def test_webpubsub_send_to_all_api_management_proxy(self, webpubsub_endpoint, webpubsub_reverse_proxy_endpoint=None):
+        client = self.create_client(endpoint=webpubsub_endpoint, reverse_proxy_endpoint=webpubsub_reverse_proxy_endpoint)
```

Comment:
I feel like there should be extra checks here, to make sure that the reverse proxy endpoint is actually doing something. This test does check that you aren't getting an error code, which is good, but I feel there should be a tests that test the `APIManagementProxy` as well, i.e., check that the client's endpoint is replaced by the different reverse proxy endpoint etc

Reply:
got it.